### DISCUSSION TITLE
eap_radius plug in in Windows

### DISCUSSION THREAD

**Discussion by psalty** [strongswan/strongswan#1580] [Q&A] [answered]
Radius could might be the most platform-independent plugin I assume.

Why strongswan doesn't support eap-radius on Windows platform?

https://docs.strongswan.org/docs/5.9/os/windows.html

**Comment by tobiasbrunner** [comment, marked as answer]
> Why strongswan doesn't support eap-radius on Windows platform?

I don't know if it's not supported. Might just not have been tested (I think the original Windows port was done for client setups, not gateways). There are certainly compilation issues, see the _1580-win-eap-radius_ branch (only compile-tested).

**Reply by psalty** [reply]
Thank you
